src/main/scala/esp/AcceleratorWrapper.scala

```diff
@@ -46,12 +46,15 @@ trait AcceleratorWrapperIO { this: RawModule =>
   // Number of 32-bit words to be read This can be converted to number of Bytes, but it was convenient since we design
   // accelerators in SytemC
   val dma_read_ctrl_data_length = IO(Output(UInt(32.W)))
+  // Size of the data tokens encoded according to AXI bus standard (e.g. double-word, word, half-word, byte).
+  val dma_read_ctrl_data_size = IO(Output(UInt(3.W)))
```

Comment:
```suggestion
  val dma_read_ctrl_data_size = IO(Output(DmaSize.gen))
```

Reply:
Thanks for the edit.

You are right: no need to change the simulated DMA.

Later we may add a feature to the testbench: we could let the testbench call the accelerator twice, emulating first the little endian and then the big endian scenarios. This would confirm that the accelerator is setting the appropriate DMA data size on every transaction.